app/models/mixins/ownership_mixin.rb

```diff
@@ -66,8 +66,14 @@ def conditions_for_owned_or_group_owned(user_or_group = nil)
       end
     end
 
-    def user_or_group_owned(user = nil)
-      where(conditions_for_owned_or_group_owned(user)).to_a
+    def user_or_group_owned(user, miq_group)
+      if user && miq_group
+        where("evm_owner_id" => user.id).or(where("miq_group_id" => miq_group.id))
+      elsif user
+        where("evm_owner_id" => user.id)
+      elsif miq_group
+        where("miq_group_id" => miq_group.id)
+      end
```

Comment:
old code was basically `where("evm_owner_id" => User.current_use.id`

Add that back in?